shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -9,86 +9,106 @@
+ * - segmentstore.cache: cache-related metrics (RocksDB)
+ * - controller.stream: metrics for operations on streams (e.g., number of streams created)
+ * - controller.segments: metrics about segments, per stream (e.g., count, splits, merges)
+ * - controller.transactions: metrics related to transactions (e.g., created, committed, aborted)
+ * - controller.retention: metrics related to data retention, per stream (e.g., frequency, size of truncated data)
+ */
+
+
 public final class MetricsNames {
     // Metrics in Segment Store Service
     // Segment-related stats
-    public static final String SEGMENT_CREATE_LATENCY = "segment_create_latency_ms"; // Timer
-    public static final String SEGMENT_READ_LATENCY = "segment_read_latency_ms";     // Timer
-    public static final String SEGMENT_WRITE_LATENCY = "segment_write_latency_ms";   // Timer
-    public static final String SEGMENT_READ_BYTES = "segmentstore.segment_read_bytes";            // Dynamic Counter
-    public static final String SEGMENT_WRITE_BYTES = "segmentstore.segment_write_bytes";          // Dynamic Counter
-    public static final String SEGMENT_WRITE_EVENTS = "segmentstore.segment_write_events";        // Dynamic Counter
+    public static final String SEGMENT_CREATE_LATENCY = "segmentstore.segment.create_latency_ms"; // Timer
+    public static final String SEGMENT_READ_LATENCY = "segmentstore.segment.read_latency_ms";     // Timer
+    public static final String SEGMENT_WRITE_LATENCY = "segmentstore.segment.write_latency_ms";   // Timer
+    public static final String SEGMENT_READ_BYTES = "segmentstore.segment.read_bytes";            // Dynamic Counter
+    public static final String SEGMENT_WRITE_BYTES = "segmentstore.segment.write_bytes";          // Dynamic Counter
+    public static final String SEGMENT_WRITE_EVENTS = "segmentstore.segment.write_events";        // Dynamic Counter
 
     //storage stats
-    public static final String STORAGE_READ_LATENCY = "tier2_read_latency_ms";   // Timer
-    public static final String STORAGE_WRITE_LATENCY = "tier2_write_latency_ms"; // Timer
-    public static final String STORAGE_READ_BYTES = "tier2_read_bytes";          // Counter
-    public static final String STORAGE_WRITE_BYTES = "tier2_write_bytes";        // Counter
+    public static final String STORAGE_READ_LATENCY = "segmentstore.storage.read_latency_ms";   // Timer
+    public static final String STORAGE_WRITE_LATENCY = "segmentstore.storage.write_latency_ms"; // Timer
+    public static final String STORAGE_READ_BYTES = "segmentstore.storage.read_bytes";          // Counter
+    public static final String STORAGE_WRITE_BYTES = "segmentstore.storage.write_bytes";        // Counter
 
     //Cache (RocksDB) stats
-    public static final String CACHE_INSERT_LATENCY = "cache_insert_latency";
-    public static final String CACHE_GET_LATENCY = "cache_get_latency";
+    public static final String CACHE_INSERT_LATENCY = "segmentstore.cache.insert_latency";
```

Comment:
since you did this, would you mind tagging the `_ms` suffix to the `latency` metrics that are missing it? Thx!